File: backend/VocabuAI.Api/Dtos/FlashCards/FlashCardImportResultDto.cs
namespace VocabuAI.Api.Dtos.FlashCards;

// Keep in sync with frontend/src/domain/dtos/flashcards/FlashCardImportResultDto.ts
public sealed record FlashCardImportResultDto(
    int ImportedFlashcardsCount,
    int ImportedLearningStatesCount,
    List<string> Errors
);

File: frontend/src/domain/dtos/flashcards/FlashCardImportResultDto.ts
// Keep in sync with backend/VocabuAI.Api/Dtos/FlashCards/FlashCardImportResultDto.cs
export type FlashCardImportResultDto = {
  importedFlashcardsCount: number;
  importedLearningStatesCount: number;
  errors: string[];
};
